astro/src/content/quickstarts/quickstart-android-kotlin-native.mdx

```diff
@@ -0,0 +1,175 @@
+import DockerSpinup from '/src/components/quickstarts/DockerSpinup.astro';
+import LoginArchitectureWeb from '/src/components/quickstarts/LoginArchitectureWeb.astro';
+import NextSteps from '/src/components/quickstarts/NextSteps.astro';
+import {RemoteCode} from '@fusionauth/astro-components';
+import RemoteContent from 'src/components/RemoteContent.astro';
+import QuickstartTshirtCTA from '/src/components/quickstarts/QuickstartTshirtCTA.astro'
+
+This quickstart is built on top of [FusionAuth Android SDK](/docs/sdks/android-sdk) demonstrating the different functionalities of the SDK available.
+
+<Intro fulltext="an Android app with Kotlin"
+    repositoryUrl="https://github.com/FusionAuth/fusionauth-quickstart-kotlin-android-native"/>
+
+## Prerequisites
+
+<RemoteContent url="https://raw.githubusercontent.com/FusionAuth/fusionauth-quickstart-kotlin-android-native/main/README.md"
+    tags="forDocSitePrerequisites" />
+
+## General Architecture
+
+<LoginArchitectureWeb/>
+
+## Getting Started
+
+In this section, you'll get FusionAuth up and running and use `git` to create a new application.
+
+### Clone the Code
+
+First off, grab the code from the repository and change into that directory.
+
+```shell
+git clone https://github.com/FusionAuth/fusionauth-quickstart-kotlin-android-native.git
+cd fusionauth-quickstart-kotlin-android-native
+```
+
+### Run FusionAuth via Docker
+
+<DockerSpinup kickstartUri={frontmatter.codeRoot + "/kickstart/kickstart.json"} />
+
+### Create your Android App
+
+Open Android Studio and select `New Project`. Choose `No Activity` and click `Next`.
+
+You can set `Name`, `Package name` and `Save location` as per your preference.
+Keep the `Language` as `Kotlin`, `Minimum API level` as `API 24 ("Nougat"; Android 7.0)` and `Build configuration language` as `Kotlin DSL`.
+
+Click `Finish`.
+
+Wait until Android Studio has finished creating and indexing the project.
+
+Add the FusionAuth SDK as a dependency to your project by changing the `app/build.gradle.kts` file. You can replace the entire file with below contents.
+
+<RemoteCode url="https://raw.githubusercontent.com/FusionAuth/fusionauth-quickstart-kotlin-android-native/main/complete-application/app/build.gradle.kts"
+            lang="kotlin"/>
+
+Replace `app/src/main/AndroidManifest.xml` as well.
+
+<RemoteCode url="https://raw.githubusercontent.com/FusionAuth/fusionauth-quickstart-kotlin-android-native/main/complete-application/app/src/main/AndroidManifest.xml"
+            lang="kotlin"/>
+
+## Authentication
+
+We'll use the [FusionAuth Android SDK](/docs/sdks/android-sdk) , which simplifies integrating with FusionAuth and creating a secure web application.
+
+### Configure FusionAuth
+
+Create `app/src/main/res/raw/fusionauth_config.json` to use the values provisioned by Kickstart.
+
+<RemoteCode url="https://raw.githubusercontent.com/FusionAuth/fusionauth-android-sdk/main/app/src/main/res/raw/fusionauth_config.json"
```

Comment:
in a real example this would be likely even a more complex configuration with a variable which builds an app against the dev, test and prod fusionauth server. Not sure if we need to explain this in more detail in the quickstart.